Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml

```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```

Comment:
ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

Reply:
done